extensions/wikia/CommunityPage/CommunityPageSpecialHooks.class.php

```diff
@@ -14,4 +14,63 @@ public static function getHTMLBeforeWikiaPage( &$html ) {
 		}
 		return true;
 	}
+
+	/**
+	 * Cache key invalidation when an article is edited
+	 *
+	 * @param $article
+	 * @param $user
+	 * @param $text
+	 * @param $summary
+	 * @param $minoredit
+	 * @param $watchthis
+	 * @param $sectionanchor
+	 * @param $flags
+	 * @param $revision
+	 * @param $status
+	 * @param $baseRevId
+	 * @return bool
+	 */
+	public static function onArticleSaveComplete(
+		&$article, &$user, $text, $summary, $minoredit, $watchthis,
```

Comment:
No need for the `&` on `$article` and `$user` as objects are effectively passed by reference in PHP anyway.